react-toggle-switch-component/my-app/src/ToggleSwitch.js

```diff
@@ -0,0 +1,28 @@
+import { useState } from "react";
+import './ToggleSwitch.css';
+
+export default function ToggleSwitch() {
+ const [isOnOff, setIsOnOff] = useState(false)
```

Comment:
`isOnOff` is a descriptive name, but it is hard to read. If the value is true, is the toggle on or off? If the value is false, is the toggle on or off? Just `isOn` or `isOff` would be easier to understand.

Reply:
Makes sense.  isOn(false) actually means something (off)